docs/remote/ssh.md

```diff
@@ -143,7 +143,7 @@ Finally, you'll be asked to pick a config file to use. You can also set the `"re
 
 For example, entering `ssh -i ~/.ssh/id_rsa-remote-ssh yourname@remotehost.yourcompany.com` in the input box would generate this entry:
 
-```sshconfig
+```ssh-config
```

Comment:
This might not work looking at the vscode-docs CI build log.
"WARNING -- WILL NOT HIGHLIGHT CODE BLOCK (```) DUE TO ERROR :::No language registration for ssh-config"